### PR TITLE
VTCCA-3023: make sure the persisted evidenceType is reloaded in the form

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

**Comment by jon-pike** [comment]
👍

File: app/controllers/propertyLinking/UploadController.scala [hunk 1]
```diff
@@ -43,7 +43,7 @@ class UploadController @Inject()(
                                   val errorHandler: CustomErrorHandler,
                                   authenticatedAction: AuthenticatedAction,
                                   withLinkingSession: WithLinkingSession,
-                                  businessRatesAttachmentsServices: BusinessRatesAttachmentService
+                                  businessRatesAttachmentsService: BusinessRatesAttachmentService
```
Comment (PeterPerhac) [inline app/controllers/propertyLinking/UploadController.scala]: i think now the name of the parameter is good `businessRatesAttachmentsService` but the name of the service should be plural "Attachments" not "Attachment" (BusinessRatesAttachment**s**Service)

Comment (habeeb01) [reply app/controllers/propertyLinking/UploadController.scala]: cool, will do that. Will do the same for the connector as well

**Comment by reddy-yattapu** [comment]
:+1:

**Pull request merged**